### PR TITLE
Add excludeGroupInfo function

### PR DESCRIPTION
**Pull request by Powersource**
For https://github.com/ssbc/ssb-tribes2/pull/88

### CODE REVIEW

File: test/tribes.js
Comment (Powersource) [inline test/tribes.js:362]: todo

**Comment by Powersource** [comment]
@SocketSecurity ignore

**Comment by Powersource** [comment]
@SocketSecurity ignore-all

File: test/tribes.js
Comment (mixmix) [inline test/tribes.js:364]: same question as other PR - is this here still so you can know you were excluded?

**Review by mixmix — APPROVED** [review]
NICE

ready to go after that other module published

**Review by staltz — APPROVED** [review]
This PR looks simple and easy, we just need to address the same things as said in ssb-keyring:

- `opts` not needed for exclude API
- `list()` shouldn't (?) have excluded group IDs
- Remember to update package.json to use the new version of keyring before merging here

Other than that, no need to ask for another review from me, looks good.

File: test/tribes.js
Comment (Powersource) [reply test/tribes.js:364]: yes that's part of the anti-gaslighting requirement: people should know that they got removed from the group, without wondering if they were in it in the first place

File: package.json
Comment (Powersource) [reply package.json:21]: nope that doesn't work with pnpm + gitlab last i tried (or maybe i didn't try `gitlab:` but i tried like literally everything else)

**Pull request merged**
